src/ShallowWrapper.js

```diff
@@ -98,6 +98,17 @@ function validateOptions(options) {
   }
 }
 
+
+function performBatchedUpdates(wrapper, fn) {
+  const renderer = wrapper.root.renderer;
+  if (renderer.unstable_batchedUpdates) {
```

Comment:
(per https://github.com/airbnb/enzyme/pull/876#issuecomment-292875234) yes, I think `REACT155` should probably be `>= 15.5` instead.